droid-command-line/src/main/java/uk/gov/nationalarchives/droid/command/action/CommandLineParam.java

```diff
@@ -315,7 +311,7 @@ public DroidCommand getCommand(CommandFactory commandFactory, CommandLine cli) {
         }
     },
     /** Open webarchive types. */
-    WEB_ARCHIVE_TYPES("Wt", "open-webarchive-types", true, 2, I18N.WEB_ARCHIVE_TYPES_HELP, "web archive types") {
+    WEB_ARCHIVE_TYPES("Wt", "open-webarchive-types", false, 2, I18N.WEB_ARCHIVE_TYPES_HELP, "web archive types") {
```

Comment:
Feel free to ignore this but is it worth putting a test in to say that the `argsRequired` is now false. There is a `CommandLineParamTest` class but there aren't many tests in it. Is it worth adding a couple of tests for these two?

Reply:
Well spotted. We use the GNU parser which throws an exception if required argument is true. As the test code is using the parser directly (i.e. not passing it to the implementation for parsing) there is not much point in testing the parameter change. The desired result is covered by the other tests.  
The problem that the test code is using the parser directly is beyond the scope of this ticket

Reply:
Ok, makes sense, thanks.